### PR TITLE
gui test: skip tests using UIActionSimulator on wxPython 4.0.7

### PR DESCRIPTION
**Pull request by pieleric**
On wxPython 4.0.7, the simulator seems to only work when it's run from
within Eclipse. From the terminal the tests always fail, because the
simulator doesn't actually do anything.

Should be working when switching to wxPython 4.1

### CODE REVIEW

File: src/odemis/gui/test/comp_text_test.py [hunk 1]
```diff
@@ -88,13 +90,22 @@ def test_unit_float(self):
...
+        wx_ver = tuple(int(v) for v in wx.__version__.split("."))
+        if wx_ver < (4, 1, 0):
+            self.skipTest("Test case is known to fail on wxPython < 4.1 due to buggy UIActionSimulator")
+
     def test_int_txt_ctrl(self):
 
         ctrl = IntegerTextCtrl(self.panel, value=123456789)
         self.add_control(ctrl, label=ctrl.__class__.__name__, flags=wx.EXPAND | wx.ALL)
 
-        test.goto_manual()
```
Comment (tepals) [inline src/odemis/gui/test/comp_text_test.py:96]: why did you remove this?

Comment (pieleric) [reply src/odemis/gui/test/comp_text_test.py:96]: It's already done for the whole file at line 35. So this line has no effect.
Actually, I think we could probably change the `gui.test` so that this function is automatically done whenever it's imported (it just read an environment variable). But that's for another PR.
